carma_wm/include/carma_wm/SignalizedIntersectionManager.hpp

```diff
@@ -144,6 +155,69 @@ class SignalizedIntersectionManager
   */
   lanelet::Lanelets identifyInteriorLanelets(const lanelet::Lanelets& entry_llts, const std::shared_ptr<lanelet::LaneletMap>& map);
 
+  /**
+   * @brief processSpatFromMsg update map's traffic light states with SPAT msg
+   *
+   * @param spat_msg Msg to update with
```

Comment:
Can we complete this description now that we are updating the code here? And also add the second parameter?

Reply:
Updated